README.md

```diff
@@ -1 +1,237 @@
+})();
+~~~
+
+> Note: the `collectFaucetTokens` method only works under development environment
+> and for accounts with not enough tokens to create new processes.
+
+### Creating a voting process
+
+After you successfully registered your account against Vocdoni's blockchain, you
+can start creating processes.
+
+A process will require a census of people who will vote it. Let's start creating
+it:
+
+~~~ts
+const census = new PlainCensus()
+census.add(address)
+// "address" is a valid ethereum-alike address
+~~~
+
+When creating the process, we recommend you adding yourself to the census:
+
+~~~ts
+const census = new PlainCensus()
+// signers return addresses as promises, hence the usage of async here
+(async () => {
+  census.add(await signer.getAddress())
+})();
+~~~
+
+After you got all the addresses for your census, you may as well create the
+process instance:
+
+~~~ts
+const election = new Election({
+  title: 'Election title',
+  description: 'Election description',
+  // a header image for your process (this is for example purposes; avoid using random sources)
+  header: 'https://source.unsplash.com/random/2048x600',
+  startDate: new Date('2023-01-01 00:00:00'),
+  endDate: new Date('2023-01-23 23:23:23'),
+  census,
+})
+~~~
+
+Of course you will need some questions in this voting process, how would people
+vote otherwise?
+
+~~~ts
+election.addQuestion('Ain\'t this process awesome?', [
+  {
+    title: 'Yes',
+    value: 0,
+  },
+  {
+    title: 'No',
+    value: 1
+  }
+])
+~~~
+
+> If you're a developer, maybe the value set to zero in Yes (and viceversa)
+> confuses you. Note that this is a mapping of values; people voting on Yes will
+> properly set the value as specified (zero in this case), thus printing the
+> results as you expect.
+
+You can finally confirm the transaction in the blockchain by just calling
+`createElection`:
+
+~~~ts
+(async () => {
+  const id = await client.createElection(election)
+  console.log(id) // will show the created election id
+})();
+~~~
+
+The election id you got there will be the one you use to access the election.
+After a few minutes of creating it, you should be able to check it on
+[our explorer][vochain explorer] (or the [dev one][dev vochain explorer] if
+you're using the development environment).
+
+### Fetching process info
+
+You can always access a process information and metadata using `fetchElection`:
+
+~~~ts
+(async () => {
+  client.setElectionId(id)
+  const info = await client.fetchElection()
+  console.log(info) // shows election information and metadata
+})();
+~~~
+
+See the [Election interface][election interface] details for more information
+about the returning object.
+
+### Voting to a process
+
+To vote a process you only need two things: the process id to vote to, and the
+option (or options) being voted:
+
+~~~ts
+(async () => {
+  client.setElectionId(id)
+  // voting yes, since it's the value of the only option
+  const vote = await client.submitVote([0] as any)
```

Comment:
There is the `Vote` object, I would avoid the `any` usage. Probably that's because I used this for some testing at some point, but the idea is to use the corresponding object. I have to check if it's exported by the library because maybe that's my fault.

Reply:
Yeah, it's due to the fact I used the example you gave me some weeks ago. I'll change this too.